js/script.js

```diff
@@ -0,0 +1,50 @@
+var url = 'https://restcountries.eu/rest/v1/name/';
+var countriesList = $('#countries');
+
+$('#search').click(searchCountries);
+
+function searchCountries() {
+ 	var countryName = $('#country-name').val();
+  if(!countryName.length) countryName = 'Poland';
+  $.ajax({
+  		url: url + countryName,
+  		method: 'GET',
+  		success: showCountriesList
+  	});
+  }
+
+var flag = "https://restcountries.eu/data/col.svg"
+
+function showCountriesList(resp) {
+  countriesList.empty();
+  resp.forEach(function(item) {
```

Comment:
before you use forEach you should be sure that resp is an array with at least one element

Reply:
if(resp)